p2p/peer.go

```diff
@@ -532,7 +534,8 @@ func createMConnection(
 			}
 		}
 		p.metrics.PeerReceiveBytesTotal.With(labels...).Add(float64(len(msgBytes)))
-		p.metrics.MessageReceiveBytesTotal.With("message_type", p.mlc.ValueToMetricLabel(msg)).Add(float64(len(msgBytes)))
+		p.metrics.MessageReceiveBytesTotal.With("message_type",
+			p.mlc.ValueToMetricLabel(msg), "chID", fmt.Sprintf("%#x", chID)).Add(float64(len(msgBytes)))
```

Comment:
[no changes needed]
using fmt.Sprintf in a heavy path is something to keep an eye on in the future

https://github.com/tendermint/tendermint/pull/8845#issuecomment-1164483127